docs/zero_1.md

```diff
@@ -0,0 +1,204 @@
+<!--- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!--- SPDX-License-Identifier: Apache-2.0  -->
+
+# What is ZeRO-1 and How Can You Use It
+
+In this document, we discuss ZeRO-1 and how to use it in Ratex.
+
+As models increase in size, speeding up training becomes increasingly difficult. Typically, data-distributed parallelism (DDP) is one of the simplest ways to reduce training time through parallelization. It works by separating batches of data across multiple processors to be processed independently with gradients being summed afterwords. However, training a model like so implies that each parallel worker stores a copy of the entire model locally. Due to the memory limitations of GPU’s, this poses a strong limit on the size of a model that can be trained efficiently using DDP.
+
+ZeRO is a form of data parallelism that works to reduce peak GPU memory consumption by partitioning model states across the parallel workers. By doing so it offers the potential to train models several magnitudes larger than with simple data distributed parallelism.
+
+ZeRO-1 is the first stage of ZeRO, which specifically partitions the optimizer states. Below, we discuss how to use ZeRO-1 to parallely train your model through an example.
+
+## 1. Ratex Set-Up
```

Comment:
We can skip this section here since we have the installation guide in the QUICKSTART

Reply:
Settled offline, leaving Ratex setup instructions to include RAF build configurations.